AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_physx_collider.py

```diff
@@ -0,0 +1,499 @@
+
+
+class EditorPhysxCollider:
+    """
+    PhysxCollider class is used to interact with the PhysX Collider Entity Component and its properties. This class
+        makes it easy for the user to interact with the PhysX Collider in a natural and easy-to-write/maintain way.
+    The PhysxCollider object stores a reference to an EditorComponent and is created by passing an EditorEntity to the
+    PhysxCollider object constructor PhysxCollider(EditorEntity).
+    """
+    def __init__(self, editor_entity: EditorEntity) -> None:
+        self.component = editor_entity.add_component(PHYSX_COLLIDER)
+
+    class Path:
+        """
+        A container class for the PropertyTree paths relevant to the PhysX Collider. This tree can be returned by
+            calling self.component.get_property_type_visibility() which will also return the expected data type
+            and visibility.
+        """
+        class Box:
+            BASE = 'Shape Configuration|Box'
+            DIMENSIONS = 'Shape Configuration|Box|Dimensions'
+        class Sphere:
+            BASE = 'Shape Configuration|Sphere'
+            RADIUS = 'Shape Configuration|Sphere|Radius'
+        class Capsule:
+            BASE = 'Shape Configuration|Capsule'
+            HEIGHT = 'Shape Configuration|Capsule|Height'
+            RADIUS = 'Shape Configuration|Capsule|Radius'
+        class Cylinder:
+            BASE = 'Shape Configuration|Cylinder'
+            RADIUS = 'Shape Configuration|Cylinder|Radius'
+            HEIGHT = 'Shape Configuration|Cylinder|Height'
+            SUBDIVISION = 'Shape Configuration|Cylinder|Subdivision'
+        class PhysicsAsset:
+            BASE = 'Shape Configuration|Asset'
+            PHYSX_MESH = 'Shape Configuration|Asset|PhysX Mesh'
+            ASSET_SCALE = 'Shape Configuration|Asset|Configuration|Asset Scale'
+            PHYSICS_MATERIALS_FROM_ASSET = 'Shape Configuration|Asset|Configuration|Physics materials from asset'
+
+        COLLISION_LAYER = 'Collider Configuration|Collision Layer'
+        COLLIDES_WITH = 'Collider Configuration|Collides With'
+        TRIGGER = 'Collider Configuration|Trigger'
+        SIMULATED = 'Collider Configuration|Simulated'
+        IN_SCENE_QUERIES = 'Collider Configuration|In Scene Queries'
+        OFFSET = 'Collider Configuration|Offset'
+        ROTATION = 'Collider Configuration|Rotation'
+        PHYSX_MATERIAL_ASSET = ']|'  # o3de/o3de#12503 Needs a better property path
+        TAG = 'Collider Configuration|Tag'
+        RESET_OFFSET = 'Collider Configuration|Rest offset'
+        CONTACT_OFFSET = 'Collider Configuration|Contact offset'
+        DRAW_COLLIDER = 'Debug draw settings|Draw collider'
+        SHAPE = 'Shape Configuration|Shape'
+
+    # General Properties
+    # o3de/o3de#12632 - Figure out how to set dropdowns for Collison Layer and Collides With
+
+    def set_is_trigger(self, value: bool) -> None:
+        """
+        Property Type, Default Visibility - ('bool', 'Visible')
```

Comment:
I'm not sure what this docstring line means exactly. is that for a different method?
does this mean sefl.Path.TRIGGER is type bool and should be 'Visible'?

Reply:
Yes, it's an informational about the property's value type and whether or not it should be set to visible on initial setup.